### PR TITLE
Add specs for Updown::Check

### PR DESCRIPTION
**Pull request by FabienChaynes**
This PR adds specs for the `Updown::Check` class.
It uses `RSpec` as the test framework and `VCR` to record HTTP interactions.

I tested use cases described in the README so it actually kind of looks like integration tests I guess?
I'm not really sure about what would be the cleanest way to organize the files so I'm open to suggestion :slightly_smiling_face:

### CODE REVIEW

File: spec/updown/updown_check_spec.rb [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            'timings' => anything,
+            'requests' => anything,
+            'host' => hash_including('city'=>'Los Angeles', 'country'=>'US')
+          }
+        )
+      end
+    end
+  end
+
+  describe '#create' do
```
Comment (askehansen) [inline spec/updown/updown_check_spec.rb]: create is a class method so maybe it should be named ".create" instead of "#create"?

**Comment by askehansen** [comment]
Nice work! It looks good, i just think we should change "#create" to ".create" :)

**Comment by FabienChaynes** [comment]
You're definitely right, I amended the previous commit and changed "#create" to ".create" :slightly_smiling_face:
I also renamed the file from `updown_check_spec.rb` to `check_spec.rb`. I think it makes more sense because it's already in the `updown` folder.

**Comment by askehansen** [comment]
Merged. Thanks!

**Comment by jarthod** [comment]
Nice, thanks a lot @FabienChaynes!

**Pull request merged**